### ISSUE TITLE
Installation Help Error

### ISSUE THREAD

**Issue by craigcecil** [SSAgov/ANDI#100]
John -- When I tried to install ANDI using the instructions on the Install page (https://www.ssa.gov/accessibility/andi/help/install.html), I ran into an issue.

Since I am using the Chromium browser on Linux, under the 'Installation Help' section, I selected the Chrome option. Step 5 there says to "Copy and paste the ANDI JavaScript URL:". However, there is no JavaScript URL visible. When I inspected that element in the DevTools to make sure it just wasn't hidden, I didn't see any text object for the URL either.

I like how you have accessible :-) step-by-step instructions for installing the bookmarklet, but it would also be nice to have a simple drag and drop install of it as well, for folks who can/want to do it that way.

Craig

**Comment by JohnCotterSSA** [comment]
Hi Craig,

Without having access to a Linux system, here's some guesses:

Maybe refresh the install page? There were some somewhat recent updates. You should see a drag and drop button that looks like this: 
![image](https://user-images.githubusercontent.com/55763887/115251322-6f071200-a0f8-11eb-92f3-14b943fdcc95.png)

Down in the installation help page, there should be a textarea box with the javascript url in it that you should be able to copy and paste. It looks like this:
![image](https://user-images.githubusercontent.com/55763887/115251784-dcb33e00-a0f8-11eb-9f15-fbfaa6a993db.png)

If that doesn't work for you, here's the url: 
`javascript:void((function(){andiScript=document.createElement('script');andiScript.setAttribute('src','https://www.ssa.gov/accessibility/andi/andi.js');document.body.appendChild(andiScript)})());`

**Comment by craigcecil** [comment]
Yeah, I had tried clearing my browser's cache, but still don't see the textarea with the code or the even the entire 'Get ANDI' section that would allow me to drag and drop the bookmarklet. I just tried Firefox on Linux and I see a completely different install page--the one you partially show above. It does have the drag-and-drop bookmarklet and the textarea with the code. Very strange. Must be something different between Chromium and Chrome.

In any case, thanks for answering and providing the javascript snippet to paste.
